spring-transaction/src/test/kotlin/org/jetbrains/exposed/spring/SpringTransactionManagerTest.kt

```diff
@@ -71,7 +71,7 @@ class SpringTransactionManagerTest {
         val tm = SpringTransactionManager(ds1)
         tm.executeAssert()
 
-        assertTrue(con1.verifyCallOrder("setReadOnly", "setAutoCommit", "commit", "close"))
+        assertTrue(con1.verifyCallOrder("setAutoCommit", "commit", "close"))
```

Comment:
These Spring tests use a `DataSourceSpy` to mock connections and save calls to `setReadOnly()` as part of transaction connection calls.

The test output has changed since readOnly is no longer unnecessarily set on every connection, since its value never changes.